lib/gds_api/config.rb

```diff
@@ -0,0 +1,17 @@
+module GdsApi
+  def self.configure
+    yield(config)
+  end
+
+  def self.config
```

Comment:
I don't think it's helpful to allow this setting per-client. That would only increase the complexity. You would configure this in an initializer, there's no reason to change this anywhere in the app.